Xamarin.Forms.Platform.Android/AppCompat/FormsAppCompatActivity.cs

```diff
@@ -115,26 +114,17 @@ protected void LoadApplication(Application application)
 			}
 
 			_application = application ?? throw new ArgumentNullException(nameof(application));
-			(application as IApplicationController)?.SetAppIndexingProvider(new AndroidAppIndexProvider(this));
+			(application as IApplicationController).SetAppIndexingProvider(new AndroidAppIndexProvider(this));
```

Comment:
direct cast would be ok too
```suggestion
			((IApplicationController)application).SetAppIndexingProvider(new AndroidAppIndexProvider(this));
```